lnrpc/lightning.proto

```diff
@@ -2000,6 +2015,14 @@ message CloseChannelRequest {
     //
     // NOTE: This field is only respected if we're the initiator of the channel.
     uint64 max_fee_per_vbyte = 7;
+
+    /*
+    A manual fee rate set in sat/kweight that should be used when crafting the
+    transaction.
+
+    NOTE: This field is only respected if we're the initiator of the channel.
+    */
+    uint64 sat_per_kweight = 8;
```

Comment:
Deprecate old field?

Reply:
you mean instead of the `sat_per_byte` use `sat_per_kweight` ?

Reply:
yes